src/azure-cli/azure/cli/command_modules/cdn/tests/latest/recordings/test_cdn_waf_policy_crud.yaml

```diff
@@ -6718,65 +4063,6 @@ interactions:
     status:
       code: 200
       message: OK
-- request:
```

Comment:
Why are there so many recording deleted?

Reply:
I believe this is due to some change in the cli core, it seems like the authentication request at the beginning of some tests is no longer needed.